workflow-examples/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- lshannon
```

Comment:
- RichardW98
- anludke

Should be here.

Reply:
Done